src/app/home/home.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-home',
+  templateUrl: './home.component.html',
+  styleUrls: ['./home.component.css']
```

Comment:
Looks like `home.component.css` is empty. What do you think about removing this line and the file, then adding it back whenever it's needed?

Reply:
done!